packages/app-desktop/services/autoUpdater/AutoUpdaterService.ts

```diff
@@ -24,18 +24,33 @@ export interface AutoUpdaterServiceInterface {
 }
 
 export default class AutoUpdaterService implements AutoUpdaterServiceInterface {
+	private window_: BrowserWindow;
+	private logger_: LoggerWrapper;
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	private initializedShim_: any;
+	private devMode_: boolean;
 	private updatePollInterval_: ReturnType<typeof setInterval>|null = null;
-
-	public constructor() {
+	private enableDevMode = true; // force the updater to work in "dev" mode
+	private enableAutoDownload = false; // automatically download an update when it is found
+	private allowPrerelease_ = false;
```

Comment:
Here as well - `includePreReleases`. Unless there's a good reason we should try to use consistent naming everywhere, it makes it easier to search for code afterwards